jenkins_jobs/upload_build_artifacts/script.sh

```diff
@@ -1,32 +1,35 @@
-# Create status json
+# ISO 8601 date with  minute precision
+CURRENT_DATE=$(date --utc -Iminutes)
```

Comment:
Good point.

I think nanosecond precision will suffice.
It would be very unlikely there would be collisions this way.